spartan/aztec-network/templates/boot-node.yaml

```diff
@@ -56,14 +56,15 @@ spec:
         - name: deploy-l1-contracts
           image: {{ .Values.images.aztec.image }}
           command:
-            [
-              "/bin/bash",
-              "-c",
-              "cp /scripts/deploy-l1-contracts.sh /tmp/deploy-l1-contracts.sh && \
-               chmod +x /tmp/deploy-l1-contracts.sh && \
-               source /shared/config/service-addresses && \
-               /tmp/deploy-l1-contracts.sh {{ .Values.ethereum.chainId }} \"{{ join "," .Values.validator.validatorAddresses }}\""
-            ]
+            - /bin/bash
+            - -c
+            - |
+              cp /scripts/deploy-l1-contracts.sh /tmp/deploy-l1-contracts.sh
```

Comment:
Out of curiosity, why were we copying the script elsewhere before running it?

Reply:
In #10540 I added a common script (now removed) to block for services to be avilable but I was getting permission errors. I couldn't figure out why so I just inlined the content into the pod.

Your comment above made me realise that it's probably becuase the scripts are mounted from a config map that's readonly.